docs/framework/windows-workflow-foundation/dynamic-update.md

```diff
@@ -55,7 +55,7 @@ DynamicUpdateServices.PrepareForUpdate(ab);
 ```
 
 > [!NOTE]
-> To download the sample code that accompanies this topic, see [Dynamic Update sample code](https://go.microsoft.com/fwlink/?LinkId=227905).
+> To download the sample code that accompanies this topic, see [Dynamic Update sample code](https://docs.microsoft.com/samples/browse/-samples).
```

Comment:
let's undo this change. this is not the right target for this. we'd have to find the source.

Reply:
ok!